### DISCUSSION TITLE
Redis Backups?

### DISCUSSION THREAD

**Discussion by Tobias-Bauer** [saleor/saleor#9864] [Q&A] [answered]
What is Redis used for in Saleor and should it be backed up?

**Comment by PlusA2M** [comment]
Hi @Tobias-Bauer ! As far as I know, Saleor use Redis for Webhooks buffer whenever Redis is available.

**Comment by timuric** [comment, marked as answer]
@Tobias-Bauer Redis is used for caching, it doesn't require back ups
